test/support/database_test_helper.ex

```diff
@@ -20,49 +20,51 @@ defmodule Astarte.VMQ.Plugin.DatabaseTestHelper do
   require Logger
   alias Astarte.Core.Device
   import ExUnit.Assertions
+  alias Astarte.Core.CQLUtils
+  alias Astarte.VMQ.Plugin.Config
 
   @create_test_keyspace """
-    CREATE KEYSPACE test
+    CREATE KEYSPACE  #{CQLUtils.realm_name_to_keyspace_name("test", Config.astarte_instance_id())}
```

Comment:
I think it would be better to use a module attribute here, since `Config.astarte_instance_id()` and `"test"` are constants in the tests:
```elixir
@test_keyspace CQLUtils.realm_name_to_keyspace_name("test", Config.astarte_instance_id())

@create_test_keyspace """
    CREATE KEYSPACE  #{@test_keyspace}
    ...
    """
# etc etc
```